src/Management/test/Endpoint.Test/Actuators/Health/EndpointMiddlewareTest.cs

```diff
@@ -298,8 +306,8 @@ public async Task GetStatusCode_OverrideUseStatusCodeFromResponseInConfiguration
         {
             var settings = new Dictionary<string, string?>(_appSettings)
             {
-                ["HealthCheckType"] = "down",
-                ["management:endpoints:UseStatusCodeFromResponse"] = "false"
+                { "HealthCheckType", "down" },
+                { "Management:Endpoints:UseStatusCodeFromResponse", "false" }
```

Comment:
Same there. There's no need to update all dictionary assignments to the new style in this PR, but let's not degrade the existing places.